src/pages/Cheon/Login/CheonLogin.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import './CheonLogin.scss';
+import { useNavigate } from 'react-router-dom';
```

Comment:
레펙토링 가이드의 순서대로 Import 순서를 반영해보세요!
```suggestion
import React, { useState } from 'react';
import { useNavigate } from 'react-router-dom';
import './CheonLogin.scss';
```